package.json

```diff
@@ -42,6 +44,12 @@
     "babel-core": "^5.8.25",
     "babel-loader": "^5.3.2",
     "chai": "^3.3.0",
+    "eslint": "^3.3.1",
+    "eslint-config-standard": "^5.3.5",
+    "eslint-config-standard-react": "^3.0.0",
+    "eslint-plugin-promise": "^2.0.1",
+    "eslint-plugin-react": "^6.1.1",
+    "eslint-plugin-standard": "^2.0.0",
```

Comment:
It's required by Standard: https://github.com/feross/eslint-config-standard/blob/master/package.json#L15